### ISSUE TITLE
XSS Vulnerabilities (Cross Site Scripting)

### ISSUE THREAD

**Issue by Jaaap** [sa-si-dev/virtual-select#73]
There are XSS vulnerabilities in `name`, `options/label` and `options/value`, example:
```javascript
VirtualSelect.init({
	ele: '#sample-select',
	multiple: true,
	name: 'aaa"><h1>BIG HEADER</h1>',
	options: [
		{ label: '<b>All will be bold', value: '1' },
		{ label: 'Value contains quote char', value: 'before"aaa' },
	],
});
```
This will result in the text BIG HEADER being rendered as an actual`<h1>` and all the options will be bold and selecting the last option would make the `value` of the hidden input "before" except that the input's className is removed by the `name` XSS.

These are technically 3 separate issues but i hope you don't mind me putting them in one ticket here.

**Comment by sa-si-dev** [comment]
@Jaaap thanks for letting me know it. I will fix those issues in the future release.

**Comment by sa-si-dev** [comment]
@Jaaap this issue got fixed in the [release v1.0.19](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.19)

To avoid this security issue, use `enableSecureText: true` property.
